stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/connection.bal

```diff
@@ -1,5 +1,17 @@
 package ballerina.net.http;
 
+const string HEADER_KEY_LOCATION = "Location";
+
+public enum redirectCode {
+    MULTIPLE_CHOICES_300,
+    MOVED_PERMANENTLY_301,
+    FOUND_302,
+    SEE_OTHER_303,
+    NOT_MODIFIED_304,
+    USE_PROXY_305,
+    TEMPORARY_REDIRECT_307
```

Comment:
Need to document these enum fields.

Reply:
Also, name of the enum should be `RedirectCode`

Reply:
Done